rero_ils/dojson/utils.py

```diff
@@ -351,7 +351,11 @@ def get_field_items(value):
 def build_string_from_subfields(value, subfield_selection, separator=' '):
     """Build a string parsing the selected subfields in order."""
     items = get_field_items(value)
-    parts = [value for key, value in items if key in subfield_selection]
+    parts = []
+    # remove special character from string
+    for key, value in items:
+        if key in subfield_selection:
+            parts.append(value.replace('\u0098', '').replace('\u009C', ''))
```

Comment:
Maybe interesting to create a `cleanValue` (in an utils.py file ?) method to remove special characters (including \u0098 and \u009C) ?